test-real-device.sh

```
@@ -0,0 +1,591 @@
+# =============================================================================
+
+set -euxo pipefail
+
+# =============================================================================
+# Configuration
+# =============================================================================
+
+ADB="/var/home/l/Android/Sdk/platform-tools/adb"
+PACKAGE="com.example.whispertoinput"
+SERVICE="com.example.whispertoinput/.WhisperInputService"
+LATIN_IME="com.android.inputmethod.latin/.LatinIME"
```

Shellcheck (0.11.0)

[warning] LATIN_IME appears unused. Verify use (or export if used externally).

(SC2034)